preprocessors/caption.schema.json

```diff
@@ -0,0 +1,14 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema",
+    "$id": "https://image.a11y.mcgill.ca/preprocessors/caption.schema.json",
+    "type": "object",
+    "title": "Caption generator",
+    "description": "Caption generated by ca.mcgill.a11y.image.preprocessor.captionGenerator",
+    "properties": {
+        "caption": {
```

Comment:
Please add a specification that the caption generated is not an empty string. As of now, `"caption": ""` would be a valid result.

Reply:
`minLength: 1` would probably make more sense. I don't expect it would ever generate something of, say, length 4, but just requiring it to be nonzero is enough in my opinion,